nipype/interfaces/dtitk/registration.py

```diff
@@ -24,35 +36,34 @@ class RigidInputSpec(CommandLineInputSpec):
                              default_value=4)
     ftol = traits.Float(mandatory=True, position=6, argstr="%s",
                         desc="cost function tolerance", default_value=0.01)
-    useInTrans = traits.Float(mandatory=False, position=7, argstr="%s",
+    useInTrans = traits.Float(position=7, argstr="%s",
                               desc="to initialize with existing xfm set as 1",
                               default_value=1)
```

Comment:
this flag (useInTrans) determines whether or not a previously calculated affine is used to initialize the registration. If it is set to anything, the affine file in the working directory with the ${prefix}.aff will be used. Otherwise, the center of mass is used to initialize. If it's a traits.Bool, setting it to false is still tripping the flag (executes as true) so I don't want to do that because it would be misleading. According to the tutorial, it should be either set to 1 or not set at all.

My thought is to have the field be traits.Int with default 1, but not specify to usedefault for any case (even the affine) so that the user must specify useInTrans=1 to initialize with an affine. Otherwise, it's undefined and the center of mass is used to initialize. Does that sound good?

Reply:
If you're not going to `usedefault=True`, setting a default value doesn't seem to have much point. And it sounds like perhaps this makes more sense as a `traits.Bool` with `argstr="1"`. That would place "1" in position 7 if True, and omit if False or unspecified. Does that seem reasonable?

Reply:
Yes. Thank you, @effigies .

Reply:
Since a rigid should be calculated before an affine, I'm setting the usedefault=True to be the case in the affine interface, but not the rigid. Does that make sense or do you think both should require the user to request that the precalculated xfm be used as the initialization?

Reply:
For the sake of flexibility, I probably would not set `usedefault` here (it becomes more difficult to un-set, so if there's ever a case you might not want the default, best to avoid), but you could use the docstring to show/describe how you expect people will want to use it.